core/src/main/scripts/importer/validateData.py

```diff
@@ -4885,24 +4890,6 @@ def main_validate(args):
     # set default to unknown because validator can be run independently from cBioPortal
     cbio_version = "unknown"
```

Comment:
It seems the default value `cbio_version` is set at two places (see previous comment). Better to choose one place where default is set.

Reply:
fixed